### PR TITLE
implemented reuse spec

### LINKED ISSUE #653: Implement REUSE spec
As already done in the react-client repo, this repo should follow the [REUSE spec](https://reuse.software/)

- [ ] Add license headers to all files
- [ ] Add licenses to `license` folder
- [ ] Add compliance check to CI

### PR DESCRIPTION
**Pull request by DerMolly**
### Component/Part
reuse

### Description
This PR adds reuse information and adds the reuse workflow

### Steps

- [x] Added implementation
- [x] I read the [contribution documentation](https://github.com/hedgedoc/hedgedoc/blob/develop/CONTRIBUTING.md) and
  signed-off my commits to accept the DCO.

### Related Issue(s)
fixes #653

### CODE REVIEW

**Review by InnayTool — DISMISSED** [review]

**Review by davidmehren — DISMISSED** [review]

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC0-1.0
+
+Files: docs/requirements.txt
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC0-1.0
+
+Files: docs/**/*.png
+Copyright: 2020 The HedgeDoc developers (see AUTHORS file)
+License: CC0-1.0
```
Comment (ErikMichelson) [inline .reuse/dep5:32]: Just asking: Images for the docs under CC0? I just thought that it would be - like the other docs content - under CC-BY-SA-4.0.

**Comment by ErikMichelson** [comment]
In general: How about adding the WebStorm copyright definition? Developers that use WebStorm (afaik currently all core-devs) get the benefit of not having to add the REUSE headers to new files by hand anymore.

File: .reuse/dep5 (hunk 1 above)
Comment (DerMolly) [reply .reuse/dep5:32]: I'll change that

Comment (DerMolly) [reply .reuse/dep5]: done

**Review by InnayTool — APPROVED** [review]

**Review by ErikMichelson — APPROVED** [review]

**Review by mrdrogdrog — APPROVED** [review]

**Pull request merged**